<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>570</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>870</width>
    <height>570</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>870</width>
    <height>570</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>870</width>
    <height>570</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>870</width>
    <height>570</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>SLOTS!</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/images/A.png</normalon>
   </iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
background-color: qradialgradient(spread:pad, cx:0.500217, cy:0.5, radius:0.5, fx:0.500101, fy:0.5, stop:0 rgba(60, 60, 60, 255), stop:0.981567 rgba(45, 45, 45, 255), stop:1 rgba(45, 45, 45, 255));
background-image: url(&quot;:/images/fullBG.png&quot;);
}

QPushButton {
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 #FFD700 , stop:1 #DAA520);
padding: 20px;
border-radius: 75px;
color:rgb(150, 110, 10);
border: 5px solid rgb(150, 110, 10);
}
QPushButton:pressed {
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 	#FFD700, stop:1 	#DAA520);
}</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="slot1">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>141</y>
      <width>150</width>
      <height>170</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/animationShadow.gif</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="slot2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>141</y>
      <width>150</width>
      <height>170</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/animationShadow.gif</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="slot3">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>141</y>
      <width>150</width>
      <height>170</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/animationShadow.gif</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonBet">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>375</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Broadway</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>BET</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPull">
    <property name="geometry">
     <rect>
      <x>481</x>
      <y>375</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Broadway</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>PULL</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWon">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>470</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <pointsize>26</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: goldenrod;</string>
    </property>
    <property name="text">
     <string>YOU WON 20 COINS!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelBet">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: antiquewhite;</string>
    </property>
    <property name="text">
     <string>BET 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMultiplier">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>310</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Relation</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  goldenrod;
</string>
    </property>
    <property name="text">
     <string>x9</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelCoins">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>500</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: antiquewhite;</string>
    </property>
    <property name="text">
     <string>60 COINS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
